src/components/AppBar/Logo.js

```diff
@@ -30,8 +32,17 @@ const Tagline = styled.p`
 `;
 
 function Logo() {
+  const { animateAndRedirect } = useContext(LandingPageAnimationContext);
+
+  function handleClick(e) {
+    e.preventDefault();
+    if (window.location.pathname !== ROOT_PATH) {
```

Comment:
Thought about it, because direct accessing `window` object is not the best way. However, I would have to modify `AppBar` component, only to pass this prop to `Logo`. Is it worth it? What if I would have 4 components?

Reply:
Anyway fixed in cbf0a95dcae12f1eb3d753d677b993a5e3ac3a77, but looking at this diff, i'm not convinced is this _better way_, if we don't plan use this app natively (iOS/Android etc..)

Reply:
The easiest way to bypass props drilling issue would be wrapping Logo component with _withRouter_ HoC.

Reply:
Thanks. That's much better - fixed in 12d099b12d056988db1f81f5635f8a030f8e5dfe